xbmc/video/Bookmark.cpp

```diff
@@ -44,3 +49,31 @@ bool CBookmark::IsPartWay() const
 {
   return totalTimeInSeconds > 0.0f && timeInSeconds > 0.0f;
 }
+
+std::string CBookmark::GetBookmarkTitle()
+{
+  switch (type)
+  {
+  case CBookmark::RESUME:
+  case CBookmark::STANDARD:
+    return StringUtils::SecondsToTimeString((long)timeInSeconds, TIME_FORMAT_HH_MM_SS);
+  default:
+    if (mediaType == MediaTypeEpisode)
+    {
+      if (seasonNumber == 0 && episodeNumber == 0)
```

Comment:
I know you didn't set the default of `seasonNumber` to `0` but `0` is a valid season number (it refers to special seasons) and `-1` refers to the `All seasons` node (but is not used in combination with episode numbers)  so maybe the default value of `seasonNumber` should be changed to `-1` and this check here adjusted as well.